### ISSUE TITLE
MiniMap Stroke Width

### ISSUE THREAD

**Issue by jparmenter** [wbkd/react-flow#862]
In the flow I'm working the background is black and made the strokeColor to match my border on my nodes. It is barely visible in the mini map. If you have a way to customize stroke width I'd appreciate it! Or can pitch in to add that prop.

<img width="1536" alt="Screen Shot 2021-01-29 at 1 43 19 PM" src="https://user-images.githubusercontent.com/389072/106325813-a45a3d00-6238-11eb-8b55-a2d10b1931b8.png">

```
      <MiniMap
        nodeStrokeColor={(n) => {
          return '#73d2d2';
        }}
        nodeColor="#4a4e4e"
      />
```

**Comment by moklick** [comment]
For now you would need to overwrite it with css. What do you think about a nodeStrokeWidth prop? We could add that.

**Comment by jparmenter** [comment]
Yeah that'd be great! 💯

**Comment by moklick** [comment]
released in v8.8.0
